opencensus-shim/src/test/java/io/opentelemetry/opencensusshim/InteroperabilityTest.java

```diff
@@ -412,28 +414,23 @@ void testSupportedMetricsExportedCorrectly() {
       statsRecorder.newMeasureMap().put(latency2, 60).record();
     }
     List<List<MetricData>> exported = metricExporter.waitForNumberOfExports(3);
-    List<MetricData> metricData = exported.get(2);
+    List<MetricData> metricData =
```

Comment:
I don't think this is supposed to relate to Guava, but somehow this test has a different order in #2586. Sorting is the sane thing to do though I think.